Formula/qpid-proton.rb

```diff
@@ -0,0 +1,38 @@
+class QpidProton < Formula
+  desc "High-performance, lightweight AMQP 1.0 messaging library"
+  homepage "https://qpid.apache.org/proton/"
+  url "https://www.apache.org/dyn/closer.lua?path=qpid/proton/0.18.0/qpid-proton-0.18.0.tar.gz"
+  sha256 "7f54c3555a8482d439759c087d656d881369252af23f209f1fd5a7e4a8c59bd7"
+  head "git://git.apache.org/qpid-proton.git"
+
+  depends_on "cmake" => :build
+  depends_on "libuv"
+  depends_on "openssl"
+
+  def install
+    ENV["OPENSSL_ROOT_DIR"] = Formula["openssl"].opt_prefix
```

Comment:
@DomT4 do you mind checking if there's some way to avoid this?

Reply:
Sure, I'll look into it shortly. About to go throw together some fajitas so I'll grab a moment whilst things are in the pan.

Reply:
I'll look into this as well at some point, but `brew pull` is 💀 if you have automatic GPG signing of git commits set up.

Reply:
can you just `ENV["HOME"] = "fuckoff"`

Reply:
I have a couple of aliases in place, `gitsoff` and `gitson` so more or less the same deal 😄.

Reply:
Does an alright job by itself locally:
```
-- Build type is "Release"
-- PN_VERSION: 0.18.0 ()
-- Can't locate the valgrind command; no run-time error detection
-- Found OpenSSL: /usr/local/opt/openssl/lib/libcrypto.dylib (found version "1.0.2m")
-- Found PythonInterp: /usr/bin/python (found version "2.7.10")
-- Found SWIG: /usr/local/bin/swig (found version "3.0.12")
-- Found CyrusSASL: /usr/lib/libsasl2.dylib
-- Looking for clock_gettime
-- Looking for clock_gettime - found
```

```
~> otool -L  /usr/local/Cellar/qpid-proton/0.18.0/lib/libqpid-proton* | grep openssl
	/usr/local/opt/openssl/lib/libssl.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libssl.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libssl.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libssl.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libssl.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libssl.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libssl.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libssl.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libssl.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
```